Bowling/BowlingGame.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Bowling.Exceptions;
+using Bowling.UtilityComponents;
+
+namespace Game
+{
+    public class BowlingGame
+    {
+        private List<Player> players = new List<Player>();
+        private List<Frame> framesScoreBoard = new List<Frame>();
+        private int CurrentFrameIndex = 0;
+        private int bonus = -1;
+        public int NumberOfPlayers { get; set; }
+        public int NumberOfFrames { get; set; }
+        public BowlingGame(int numberOfFrames, int numberOfPlayers)
+        {
+            if (numberOfPlayers < 1)
+            {
+                throw new NotEnoughPlayersException(Constants.numberOfPlayersMessage);
+            }
+            else
+            {
+                NumberOfPlayers = numberOfPlayers;
+            }
+
+            if (numberOfFrames < 1 || numberOfFrames > 10)
```

Comment:
Why do you expose the possibility of selecting the number of frames? 
In a bowling game there are only 10 frames therefore I would make this a private constant inside the Game class.